tests/integration/test_utils/data_source_config/snowflake.py

```diff
@@ -48,29 +48,30 @@ class SnowflakeConnectionConfig(BaseSettings):
     SNOWFLAKE_PW: str
     SNOWFLAKE_ACCOUNT: str
     SNOWFLAKE_DATABASE: str
-    SNOWFLAKE_SCHEMA: str
     SNOWFLAKE_WAREHOUSE: str
     SNOWFLAKE_ROLE: str = "PUBLIC"
 
     @property
     def connection_string(self) -> str:
         return (
             f"snowflake://{self.SNOWFLAKE_USER}:{self.SNOWFLAKE_PW}"
-            f"@{self.SNOWFLAKE_ACCOUNT}/{self.SNOWFLAKE_DATABASE}/{self.SNOWFLAKE_SCHEMA}"
+            f"@{self.SNOWFLAKE_ACCOUNT}/{self.SNOWFLAKE_DATABASE}"
```

Comment:
I might add a comment about how this is used and why it's ok to not have the connection string here. Namely, [we require it when configuring a snowflake datasource](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/datasource/fluent/snowflake_datasource.py#L106) so I had to read more to understand why we can not have it here.

Reply:
Good call. Updated.